### ISSUE TITLE
Tag 3.0.6 created but binaries not released

### ISSUE THREAD

**Issue by nudgegoonies** [google/mtail#900]
Maybe this is not fixed completely yet?
https://github.com/google/mtail/issues/892
Tag 3.0.6 is created: https://github.com/google/mtail/releases/tag/v3.0.6
But the releases with binaries still show v3.0.5 as highest version.

**Comment by jaqx0r** [comment]
Yep, the workflow that makes tags isn't fixed yet. Can your tool check
specifically for releases and not tags?

**Comment by nudgegoonies** [comment]
I took a look at https://github.com/lilydjwg/nvchecker and this should be possible.

**Comment by jaqx0r** [comment]
Please look at jaqx0r/mtail
